pygmt/src/rose.py

```diff
@@ -82,22 +83,22 @@ def rose(self, data=None, length=None, azimuth=None, **kwargs):
     region : str or list
         *Required if this is the first plot command*.
         *r0/r1/az0/az1*.
```

Comment:
```suggestion
        [*r0*, *r1*, *az0*, *az1*].
```

Reply:
I agree, lists are more Pythonic. Currently, `-90/90`, `0/180`, `0/360` are used in the docs. Should this be changed to `-90, 90`, `0, 180`, `0, 360`? Maybe we should state the argument structure of the `region` parameter for both _str_ and _list_  input, since slashes (`/`) are required within the string instead of commas (`,`)?

![rose_doc_str_cut](https://user-images.githubusercontent.com/94163266/188490520-175692bf-5115-4b92-80a5-47fad0dcfb89.png)

Reply:
Perhaps we can rewrite it to:
```
For *az0* and *az1*, specify either (-90, 90) or (0, 180) for half
        circle plot or (0, 360) for full circle.
```

Reply:
I made these changes in https://github.com/GenericMappingTools/pygmt/pull/2090/commits/e1a3d50965a966225ced4b4df38b7c6e95720e48 and https://github.com/GenericMappingTools/pygmt/pull/2090/commits/b5ed9e2a6c2c3d36a5c42031ee366c16ee77829b.